src/protocols/secure_channel/PASESession.cpp

```diff
@@ -57,10 +57,15 @@ const char * kSpake2pContext        = "CHIP PAKE V1 Commissioning";
 const char * kSpake2pI2RSessionInfo = "Commissioning I2R Key";
 const char * kSpake2pR2ISessionInfo = "Commissioning R2I Key";
 
+// Wait at most 2 seconds for the response from the peer.
```

Comment:
Why is this 2 seconds for PASE but 5 for CASE?

Also, see notes on the comments in the CASE bits.